packages/root-cms/core/schema.ts

```diff
@@ -57,10 +76,16 @@ export function date(field: Omit<DateField, 'type'>): DateField {
 export type DateTimeField = CommonFieldProps & {
   type: 'datetime';
   default?: string;
+  /**
+   * Timezone to use for the datetime field. If set, the field will be limited to
+   * this timezone.
```

Comment:
Provide an example of a correct tz value in the jsdoc, e.g. "America/Los_Angeles".